requirements.txt.in

```diff
@@ -0,0 +1,2 @@
+scriptworker !=13.0
+taskcluster
```

Comment:
I think scriptworker includes taskcluster as a dep, so this is extraneous. No harm though.